JPATask/src/main/java/com/instinctools/egor/mentoring/JPA/DAO/DAOFactory/Factory.java

```diff
@@ -0,0 +1,24 @@
+package com.instinctools.egor.mentoring.JPA.DAO.DAOFactory;
+
+import com.instinctools.egor.mentoring.JPA.DAO.ClientDAO;
+import com.instinctools.egor.mentoring.JPA.DAO.DAOImpl.ClientDAOImpl;
+import com.instinctools.egor.mentoring.JPA.Entity.Client;
+
+public class Factory {
```

Comment:
1) Bad naming - "Factory"
2) Any factories should lay down higher (or at least at same level) then theirs products in packages hierarchy
3) Not clear why we need a factory here at all (at least on current stage) - there is only one product, no alternatives